### ISSUE TITLE
Update Pattern stickersheets in Figma (to have variables version) - Button Bar

### ISSUE THREAD

**Issue by yunjungyeh** [jpmorganchase/salt-ds#4268]
Update Pattern stickersheets in Figma (to have variables version) - Button Bar

**Comment by bhoppers2008** [comment]
No further updates to be made to retired libraries.

@yunjungyeh please review if this is still required for Salt DS Figma.

**Comment by yunjungyeh** [comment]
Yes if we are still linking to the Pattern template file from Salt DS library

**Comment by yunjungyeh** [comment]
Reviewed, updated and completed
